### ISSUE TITLE
Adding calendar event to RSVP email

### ISSUE THREAD

**Issue by alysivji** [chicagopython/chipy.org#317]
As a user, I want to get a calendar invite when I RSVP to a meeting using ChiPy.org.

This is low priority as we are using Meetup for virtual events. When we start meeting in-person again and if we are still using ChiPy.org to get RSVPs for the main meeting, it would be nice to have this.

#### Resources

- [Email Markup](https://developers.google.com/gmail/markup/getting-started)

**Comment by elmq0022** [comment]
Also see issue #176. 

Additionay we haven't sent emails to logged in users, just the anonymous ones.  Is this something we should change? 

Could we push the site to only nonanonymous RSVPs? It would be clearner and normalize our data in the database. 

Last I looked most of the RSVPs were anonymous.

**Comment by alysivji** [comment]
I don't know what we do now or how the database is set up, I'm just thinking in terms of how I want RSVPs to work as a user (anonymous or not).

Maybe it's just me, but I find it difficult to get iCal events into my calendar: I have to download the event and import it into my calendar. Not sure how Meetup does it, but it feels like anytime I RSVP to something, it appears on my calendar and I don't miss it.

Again, this isn't going to be a problem for 6 months to a year.

**Comment by elmq0022** [comment]
Makes sense.

I didn't state what I was really thinking. That we should perhaps have a
larger project on how to handle RSVPs after leaving Meetup.

Do you think this is a good idea? Or is it more work for little benefit?

**Comment by alysivji** [comment]
My problems with our current setup:
- only the main meeting is prominently featured
- links do not unfurl nicely

I think there needs to be a wider discussion about homegrown versus meetup. I mostly created this ticket to add it to the backlog, but it's definitely something we need to think about before we start holding in-person events.
